### PR TITLE
more detailed way of how to run the tests, added readme

### PR DESCRIPTION
**Pull request by PanLuce**
more detailed way of how to run the tests, added readme for someone new to contribute to the repo, removed unnecessary stuff, introduced baseURL parameter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Prerequisites
+First you should check if you have playwright installed globally.
+
+`npm list -g`
+
+If not, then you need to install it.
+
+`npm install -g playwright`
```
Comment (vojtech-cerveny) [inline README.md:8]: Not sure if this is needed.

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "playwright test"
+    "test": "npx playwright test --reporter=list ./tests/**.spec.ts"
```
Comment (vojtech-cerveny) [inline package.json:7]: not sure if this is needed.

**Review by vojtech-cerveny — COMMENTED** [review]
we can change it in next PR due to this is already merged :) + deleted "tahak" - I wanted to generate some documentation on paper, that girls don't need to switch context in vscode and they can have some basic manual on paper.

File: README.md (hunk 1 above)
Comment (PanLuce) [reply README.md:8]: not sure either now, but I needed to do this at some point when I was dealing with new macbook

File: package.json (hunk 2 above)
Comment (PanLuce) [reply package.json:7]: this one gives us more adjustable way how to run tests - I can change it back if you're against

**Pull request merged**